.github/workflows/cross-bootstrap-tools.yml

```diff
@@ -17,22 +21,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target_arch: [ amd64, aarch64 ]
-        os: [ ubuntu-22.04, ubuntu-24.04, macos-latest ]
+        target_arch:
+          - amd64
+          - aarch64
+        os: 
+          - ubuntu-24.04
+          - macos-latest
         include:
-          # TODO: both Ubuntu and macOS have bmake packages, we should try them instead of bootstrapping our own copy.
-          - os: ubuntu-22.04
```

Comment:
This also seems to eliminate ubuntu-22 from the matrix. Is that intentional?

Reply:
We'll have to ask @jrtc27 to see which ones we can drop, since she added this originally.
I think that ubuntu-20 is safe to drop, but I think she (or some other CheriBSD person) had said that they still needed to support ubuntu-22.